File: go/1051. Height Checker.go
func heightChecker(heights []int) int {
    copy := append([]int{}, heights...)
    sort.Ints(copy)
    answer := 0
    for i := 0; i < len(heights); i++ {
        if heights[i] != copy[i] {
            answer += 1
        }
    }
    return answer
}

File: java/1051. Height Checker.java
class Solution {
    public int heightChecker(int[] heights) {
        int[] copy = new int[heights.length];
        for (int i = 0; i < heights.length; i++) {
            copy[i] = heights[i];
        }
        Arrays.sort(copy);
        int answer = 0;
        for (int i = 0; i < heights.length; i++) {
            if (copy[i] != heights[i]) answer++;
        }
        return answer;
    }
}
